docs/docs/creating-and-modifying-pages.md

```diff
@@ -6,9 +6,9 @@ Gatsby makes it easy to programmatically control your pages.
 
 Pages can be created in three ways:
 
-- In your site's gatsby-node.js by implementing the API
-  [`createPages`](/docs/node-apis/#createPages)
 - Gatsby core automatically turns React components in `src/pages` into pages
+- In your site's gatsby-node.js by implementing the API
+  [`createPages`](/docs/node-apis/#createPages) (This is an advanced feature)
```

Comment:
This whole section is duplicated from "Routing". How about replacing the whole section with

> Pages are normally created automatically from any component in `src/pages`. This simple usage is covered in [Routing](/docs/routing).

> In some situations, you might need finer control of pages, to either create or modify them programmatically. This section covers such advanced usage.

That way it's clear which information is where. Otherwise "Routing" says you can read more here but there is nothing here about the automatic creation of pages from `src/pages`.